machine-learning/README.md

```diff
@@ -1,4 +0,0 @@
-
-# Microservices for Immich
-
-## Image Classifier
```

Comment:
I understand why this file and the LICENSE were deleted, but it would be good to add back new ones with some detail on how the python setup is structured.